nx.json

```diff
@@ -105,7 +105,7 @@
     "@fluentui/react-menu": { "implicitDependencies": [] },
     "@fluentui/react-monaco-editor": { "implicitDependencies": [] },
     "@fluentui/react-portal": { "tags": ["vNext"], "implicitDependencies": [] },
-    "@fluentui/react-positioning": { "implicitDependencies": [] },
+    "@fluentui/react-positioning": { "tags": ["vNext"], "implicitDependencies": [] },
```

Comment:
pls update your base against latest master - there is new functionality/fixes for migration

Reply:
Updated and reran

Reply:
checked your branch, its not up to date (maybe you forgot to push? )